docs/_config.yml

```diff
@@ -22,10 +22,11 @@ html:
   use_repository_button: true
   use_edit_page_button: true
 
-  # NOTE this gets replaced by the build infrastructure during deployment
-  extra_navbar: __TSTRAIT_VERSION__
-  extra_footer: __TSTRAIT_VERSION__
-
+  extra_footer: |
+    <div>
+    msprime __TSTRAIT_VERSION__
```

Comment:
```suggestion
    tstrait __TSTRAIT_VERSION__
```

Reply:
Thanks for your suggestion! I missed it when I made the commitment.